packages/fetchai/skills/tac_control_contract/helpers.py

```diff
@@ -42,15 +42,13 @@ def generate_good_id_to_name(nb_goods: int, contract: Contract) -> Dict[str, str
     Generate ids for things.
 
     :param nb_goods: the number of things.
+    :param contract: the instance of the contract
     :return: a dictionary mapping goods' ids to names.
     """
-
-    item_ids = contract.create_item_ids_based_on_nb_goods(
-        token_type=NFT, nb_goods=nb_goods
-    )
     max_number_of_digits = math.ceil(math.log10(nb_goods))
+    token_ids = contract.create_item_ids_based_on_nb_goods(NFT, nb_goods)
```

Comment:
I think we should reserve the keyword `create` and `get` for interactions with the deployed contract and use `generate_item_ids_based_on_nb_goods` for interactions with the API in this particular instance